task/07-yield-tasks.js

```diff
@@ -82,7 +104,16 @@ function* getFibonacciSequence() {
  *
  */
 function* depthTraversalTree(root) {
-    throw new Error('Not implemented');
+    const stack = [root]
+    let node
+
+    while(stack.length !== 0) {
```

Comment:
```suggestion
    while(stack.length) {
```
This check will do the same, but will look a bit cleaner